### ISSUE TITLE
googlemaps pom incorrect

### ISSUE THREAD

**Issue by aegis123** [TradeMe/MapMe#20]
Hi when updating to 1.0.5 for the google maps version the base dependency is not referenced correctly in the pom.xml

```XML
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>nz.co.trademe.mapme</groupId>
  <artifactId>googlemaps</artifactId>
  <version>1.0.5</version>
  <packaging>aar</packaging>
  <description>MapMe is an Android library that brings the adapter pattern to Maps, simplifying the management of markers and annotations.</description>
  <name>MapMe GoogleMaps</name>
  <url>https://github.com/TradeMe/MapMe.git</url>
  <licenses>
    <license>
      <name>MIT License</name>
      <url>http://www.opensource.org/licenses/mit-license.php</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
  <scm>
    <url>https://github.com/TradeMe/MapMe.git</url>
  </scm>
  <dependencies>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-maps</artifactId>
      <version>11.8.0</version>
    </dependency>
    <dependency>
      <groupId>com.android.support</groupId>
      <artifactId>appcompat-v7</artifactId>
      <version>27.0.1</version>
    </dependency>
    <dependency>
      <groupId>MapMe</groupId>
      <artifactId>mapme</artifactId>
      <version>unspecified</version>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-stdlib-jre7</artifactId>
      <version>1.2.31</version>
    </dependency>
  </dependencies>
</project>
```

**Comment by josh-burton** [comment]
Hi @aegis123, good catch!

@jamiesanson looks like this may have been an issue with the way the last version was published. I've just tried publishing to my local m2 repo and the pom seems correct.

Can you try to publish again?

**Comment by jamiesanson** [comment]
Hi @aegis123,

We had some issues with the Bintray gradle plugin in use for publishing. We removed the plugin in #21, and published version 1.0.6 with the correct *.pom files
